Source/Scene/BatchTable.js

```diff
@@ -0,0 +1,475 @@
+     */
+    BatchTable.prototype.setEntry = function(instanceIndex, attributeIndex, value) {
+        //>>includeStart('debug', pragmas.debug);
+        if (instanceIndex < 0 || instanceIndex >= this._numberOfInstances) {
+            throw new DeveloperError('instanceIndex is out of range.');
+        }
+        if (attributeIndex < 0 || attributeIndex >= this._attributes.length) {
+            throw new DeveloperError('attributeIndex is out of range');
+        }
+        if (!defined(value)) {
+            throw new DeveloperError('value is required.');
+        }
+        //>>includeEnd('debug');
+
+        var attributes = this._attributes;
+        var result = setEntryScratchValues[attributes[attributeIndex].componentsPerAttribute];
+        var currentEntry = this.getEntry(instanceIndex, attributeIndex, result);
+        var entryType = getEntryType(this._attributes, attributeIndex);
+        var entriesEqual = defined(entryType.equals) ? entryType.equals(currentEntry, value) : currentEntry === value;
+        if (entriesEqual) {
+            return;
+        }
+
+        var entryValue = setEntryScratchCartesian4;
+        entryValue.x = defined(value.x) ? value.x : value;
+        entryValue.y = defined(value.y) ? value.y : 0.0;
+        entryValue.z = defined(value.z) ? value.z : 0.0;
+        entryValue.w = defined(value.w) ? value.w : 0.0;
+
+        var index = 4 * attributes.length * instanceIndex + 4 * attributeIndex;
+        Cartesian4.pack(entryValue, this._batchValues, index);
+
+        this._batchValuesDirty = true;
+    };
+
+    function createTexture(batchTable, context) {
+        var dimensions = batchTable._textureDimensions;
+        batchTable._texture = new Texture({
+            context : context,
+            pixelFormat : PixelFormat.RGBA,
+            pixelDatatype : batchTable._pixelDatatype,
+            width : dimensions.x,
+            height : dimensions.y,
+            sampler : new Sampler({
+                minificationFilter : TextureMinificationFilter.NEAREST,
+                magnificationFilter : TextureMagnificationFilter.NEAREST
+            })
+        });
+    }
+
+    function updateTexture(batchTable) {
+        var dimensions = batchTable._textureDimensions;
+        batchTable._texture.copyFrom({
+            width : dimensions.x,
+            height : dimensions.y,
+            arrayBufferView : batchTable._batchValues
+        });
+    }
+
+    /**
+     * Creates/updates the batch table texture.
+     * @param {FrameState} frameState The frame state.
+     *
+     * @exception {RuntimeError} The floating point texture extension is required but not supported.
+     */
+    BatchTable.prototype.update = function(frameState) {
+        var context = frameState.context;
+        if (this._pixelDatatype === PixelDatatype.FLOAT && !context.floatingPointTexture) {
+            // TODO: We could probably pack the floats to RGBA unsigned bytes but that would add a lot CPU and memory overhead.
```

Comment:
Leave the comment, but drop the `TODO`; I don't expect that we would implement this fallback given how rare it should be.